packages/now-next/src/index.ts

```diff
@@ -875,8 +894,19 @@ export const build = async ({
       const zippingLabel = 'Compressed shared serverless function files';
       console.time(zippingLabel);
 
-      pseudoLayers.push(await createPseudoLayer(tracedFiles));
-      apiPseudoLayers.push(await createPseudoLayer(apiTracedFiles));
+      let pseudoLayer;
+      let apiPseudoLayer;
+      ({ pseudoLayer, pseudoLayerBytes } = await createPseudoLayer(
+        tracedFiles
+      ));
+      ({
+        pseudoLayer: apiPseudoLayer,
+        pseudoLayerBytes: apiPseudoLayerBytes,
```

Comment:
Why is this using a global variable `apiPseudoLayerBytes` instead of returning the variable from the `isApiPage()` function? Should it instead be `apiPseudoLayerBytes += pseudoLayerBytes`?

Reply:
Not sure I understand, `isApiPage` just checks if the page matches an API route the `apiPseudoLayerBytes` contains the size of the shared compressed API dependency sizes, `pseudoLayerBytes` contains the size of the shared compressed non-API dependency sizes